Vagrantfile

```diff
@@ -98,9 +102,13 @@ if File.exist?(env_specific_config_path)
 end
 
 # Display the main current configuration parameters
-@ui.info "Welcome to Kubernetes playground!"
+@ui.info "Welcome to Kubernetes playground, version " + K8S_PLAYGROUND_VERSION
 @ui.info "Vagrant provider : " + settings["conf"]["vagrant_provider"]
 @ui.info "Networking plugin : " + settings["ansible"]["group_vars"]["all"]["kubernetes_network_plugin"]
+if ENV['VAGRANT_LOG']=='debug' or ENV['VAGRANT_LOG']=='info'
```

Comment:
Again, we don't need this check. I'd print this only in debug, so @ui.debug

Reply:
see previous comment